backstage/plugins/knative-event-mesh-backend/src/dynamic/index.ts

```diff
@@ -0,0 +1,22 @@
+import {BackendDynamicPluginInstaller} from '@backstage/backend-dynamic-feature-service';
+import {CatalogClient} from "@backstage/catalog-client";
+
+import {KnativeEventMeshProcessor, KnativeEventMeshProvider} from '../providers';
+
+export const dynamicPluginInstaller:BackendDynamicPluginInstaller = {
```

Comment:
```suggestion
// This is mainly to provide dynamic plugin support for backstage's legacy backend. During the plugin loading phase, it'll use this dynamicPluginInstaller.
// https://github.com/backstage/backstage/blob/master/packages/backend-dynamic-feature-service/src/manager/types.ts
// It's deprecated since the legacy backend is now deprecated, but it's probably still nice to have in case someone wants to install on the legacy backend for some reason.
export const dynamicPluginInstaller:BackendDynamicPluginInstaller = {
```